pkg/handler/handler.go

```diff
@@ -40,13 +41,42 @@ import (
 )
 
 const (
-	healthCheckAddr = ":8081"
-	livenessPath    = "/healthz"
-	readinessPath   = "/readyz"
-	metricsPath     = "/metrics"
-	whHandlePath    = "/validate/v1alpha1/blueprint"
+	livenessPathEnvVar  = "LIVENESS_PATH"
+	readinessPathEnvVar = "READINESS_PATH"
+	healthPortEnvVar    = "HEALTH_PORT"
+	metricsPath         = "/metrics"
+	whHandlePath        = "/validate/v1alpha1/blueprint"
 )
 
+var (
+	livenessPath    string
+	readinessPath   string
+	healthCheckAddr string
```

Comment:
Instead of defining global variable and setting the value with `init()` does it make sense to get the value whenever required? We can have a helper function e.g `getLivenessPath()` which returns default or overridden value whenever required. Same for other variables.